saleor/core/tracing.py

```diff
@@ -35,7 +34,7 @@ def webhooks_otel_trace(
     :param payload_size: size of the payload in bytes
     """
     with tracer.start_as_current_span(
-        f"webhooks.{span_name}", kind=trace.SpanKind.CLIENT
+        f"webhooks.{span_name}", service_scope=True, kind=SpanKind.CLIENT
```

Comment:
Are we sure all data in `webhooks.*`  should be exposed to `SERVICE` tracker?

Reply:
Yes, we agreed that all the attributes are suitable for the `service` scope